proto/storage/test.proto

```diff
@@ -137,14 +137,14 @@ message TestSingleUUIDKeyStruct {
     bytes bytess = 16;
 }
 
-message TestMultiKeyStruct {
+message TestStruct {
   enum Enum {
     ENUM0 = 0;
     ENUM1 = 1;
     ENUM2 = 2;
   }
   string key1                  = 1 [(gogoproto.moretags) = 'sql:"pk,id" search:"Test Key"'];
-  string key2                  = 2 [(gogoproto.moretags) = 'sql:"pk" search:"Test Key 2,store,hidden"'];
+  string key2                  = 2 [(gogoproto.moretags) = 'search:"Test Key 2,store,hidden"'];
```

Comment:
Should we change this field name since it is no longer a key?

Reply:
I tried to remove it but then protocheck fails. I think it's OK to keep that